### ISSUE TITLE
Should we assign roles?

### ISSUE THREAD

**Issue by alexcheninfo** [why-el/crowd-written-novel#50]
I worry that the novel will get stuck in the opening sentence if we don't assign roles. For instance, ideas guys, writers, ghostwriters, editors, etc.

**Comment by murtazakhussain** [comment]
Not a bad Idea, crowd sourcing is second to stuck

**Comment by alexcheninfo** [comment]
I'm busy working on a novel right now, but I can write the first paragraphs of the selected opening sentence. And act as an editor later on (if I have time).

**Comment by Ghoughpteighbteau** [comment]
I think @why-el might be a touch overwhelmed. 

What needs to happen right now is @jfonte and @n-r need to be given write access to the repository. Others too.

If nothing happens friday, I'll start accepting pull requests and giving people write access on my fork.

Any objections?

**Comment by why-el** [comment]
@Ghoughpteighbteau Yep, that's what I was thinking as well. I'll give commit access later tonight/tomorrow. I have over 50 related emails to go over.

**Comment by Ghoughpteighbteau** [comment]
Sweet! I was not looking forward to that.

**Comment by n-r** [comment]
Thanks for adding me to the write-access list, I appreciate it very much.

**Comment by why-el** [comment]
Alright, so who is interested in becoming who whom? 

I will go over the structure issue (#6) to see if I can recruit some more people.
